cloud-consumption/src/main/java/com/sequenceiq/consumption/service/ConsumptionService.java

```diff
@@ -43,7 +43,7 @@ public class ConsumptionService extends AbstractAccountAwareResourceService<Cons
     private ConsumptionDtoConverter consumptionDtoConverter;
 
     @Inject
-    private CloudStorageLocationUtil cloudStorageLocationUtil;
+    private AwsS3ConsumptionCalculator awsS3ConsumptionCalculator;
```

Comment:
do not directly inject the bean

Reply:
What should i do instead?

Reply:
Actually we discussed it with @doktoric  to refactor it when we implement the EBS as there are lots of functions and related to aggregation which can change for other service.

Reply:
Sorry but I do not see the point. Since we already have the `CloudConnector.consumptionCalculators()` lookup as done in `SendConsumptionEventHandler`, the very same approach shall also work here. I agree that the logic might have to change later depending on the storage type, but that still does not justify directly injecting `AwsS3ConsumptionCalculator` here. Especially since it is AWS-specific and completely ignores the actual cloud platform.

Reply:
done